### ISSUE TITLE
make navbar responsive

### ISSUE THREAD

**Issue by tejasnasre** [nexmeet/nexmeet#14]
make current navbar responsive

figma link : https://www.figma.com/design/ywxfYNYaOzZgPGyZi3WQ7j/NextMeet?node-id=141-294&t=zC0VjvYMyEr1WNPW-1

**Comment by NOOB-3301** [comment]
I want to work on this . Can you assign me to it under swoc ??

**Comment by TejasNasre** [comment]
go ahead

**Comment by NOOB-3301** [comment]
can i make pr on or after 1st jan or have to do it now ??

**Comment by TejasNasre** [comment]
After Jan it will be marked so built it in local don't create PR now just post video here and move to other issues

**Comment by NOOB-3301** [comment]
https://github.com/user-attachments/assets/472e65b4-9f28-4d3b-ae4d-85e4fdb0a003

This are the some changes i made , adding some transition , responsiveness for android devices

let me know if this are ok or what should i change next ??

**Comment by IshadP** [comment]
> hyprrecording2.mp4 
> This are the some changes i made , adding some transition , responsiveness for android devices
> 
> let me know if this are ok or what should i change next ??

Overlay for navbar has been added in design, do check out for implementation

**Comment by NOOB-3301** [comment]
> > hyprrecording2.mp4
> > This are the some changes i made , adding some transition , responsiveness for android devices
> > let me know if this are ok or what should i change next ??
> 
> Overlay for navbar has been added in design, do check it out for implementation.

![hyperss-2024-12-27_13-21-20](https://github.com/user-attachments/assets/899bac0f-9348-4259-8e00-cceb1f1b3abd)
will this be okay or i have to change ???

**Comment by IshadP** [comment]
Try to stick to design. You can use prototype mode on mobile version to see how interaction works

Link to prototype:
https://www.figma.com/proto/ywxfYNYaOzZgPGyZi3WQ7j/NextMeet?page-id=24%3A138&node-id=237-210&viewport=3781%2C1681%2C0.9&t=54WFae8ntOekWt23-1&scaling=scale-down&content-scaling=fixed&starting-point-node-id=237%3A210

**Comment by TejasNasre** [comment]
@NOOB-3301 yeah stick to the design

**Comment by NOOB-3301** [comment]
![hyperss-2024-12-27_19-58-57](https://github.com/user-attachments/assets/e8074e2b-c24e-4dbf-855b-6f248c415b54)
![hyperss-2024-12-27_19-59-17](https://github.com/user-attachments/assets/ac325fa7-4194-4e75-b16a-222dc5cd6a7a)
![hyperss-2024-12-27_20-03-43](https://github.com/user-attachments/assets/72cbfa9b-13c3-440d-97c7-a4b091fb7ff6)
![hyperss-2024-12-27_20-06-05](https://github.com/user-attachments/assets/d223b59f-9338-4145-b8ff-a3dcb0342f9b)


is this is ok ??
do you guys have any discord server related to project. Like it will be better to send the update  there instead of pinging here repeatedly @TejasNasre @IshadP

**Comment by TejasNasre** [comment]
@IshadP please review and also checkout https://nexmeet.onrender.com/

**Comment by tejasnasre** [comment]
@NOOB-3301 please drop working video of issue

**Comment by NOOB-3301** [comment]
[this is the demo video](https://drive.google.com/file/d/1BAttXjnYXVEhNT3tSqdsc1HhjgNOQGbq/view?usp=drive_link) 

here is the drive link please checkit out @tejasnasre

**Comment by NOOB-3301** [comment]
I have made the pr. Do check it out . Also can you please provide SWOC label to it?

**Comment by tejasnasre** [comment]
Yeah too night I will add labels and merged it as well and will also create discord as well

**Comment by tejasnasre** [comment]
> [this is the demo video](https://drive.google.com/file/d/1BAttXjnYXVEhNT3tSqdsc1HhjgNOQGbq/view?usp=drive_link)
> 
> here is the drive link please checkit out @tejasnasre

Great will merge tonight

**Comment by NOOB-3301** [comment]
> > hyprrecording2.mp4
> > This are the some changes i made , adding some transition , responsiveness for android devices
> > let me know if this are ok or what should i change next ??
> 
> Overlay for navbar has been added in design, do check it out for implementation.

hey @IshadP  , are you working on the profile page, or you are working on the figma design. i checked #12  , if you havent done it yet i can help :+1:  . share your discord id.

**Comment by IshadP** [comment]
Any help is very much appreciated!

**Comment by tejasnasre** [comment]
bro you changed too much ui and not covered edge case please cover all cases next time

**Comment by NOOB-3301** [comment]
@IshadP send you message on in discord do check out

**Comment by tejasnasre** [comment]
@NOOB-3301 https://discord.gg/7zD5kycV
